components/nimbus/src/enrollment.rs

```diff
@@ -373,40 +376,88 @@ impl ExperimentEnrollment {
                 enrollment_id,
                 branch,
                 ..
-            } => EnrollmentChangeEvent::new(
-                &self.slug,
-                enrollment_id,
-                branch,
-                None,
-                EnrollmentChangeEventType::Enrollment,
-            ),
+            } => {
+                #[cfg(feature = "builtin-glean")]
+                nimbus_events::enrollment.record(nimbus_events::EnrollmentExtra {
+                    branch: Some(branch.to_string()),
+                    enrollment_id: Some(enrollment_id.to_string()),
+                    experiment: Some(self.slug.to_string()),
+                });
+
+                let mut extra = HashMap::new();
+                extra.insert("enrollment_id".to_string(), enrollment_id.to_string());
+
+                #[cfg(feature = "builtin-glean")]
+                glean::set_experiment_active(
+                    enrollment_id.to_string(),
+                    branch.to_string(),
+                    Some(extra),
+                );
```

Comment:
super nit: we can `cfg(feature=` a block here instead of multiple `cfg`s, i.e adding curly brackets before and after the part that is for glean-in-rust

Reply:
eeeh I relalized there is a line in between that is not cfg'ed nvm this comment!

Reply:
Yeah, I should have done that here, the extra isn't really used without Glean and I missed conditionally compiling it away anyways.  I'll update this to use the `cfg!() {}` macro and wrap the whole block of Glean related code.